english/challenge-marshmellow-to-rock-off/castle-route/narnia.md

```diff
@@ -24,3 +24,7 @@ disgust under its dark hood as it scurries to the stairs and vanishes in the dar
 cryptic language written on the hilt of the sword, you need to decide.
 
 Will it be the ladder? or the Stairs?
+
+[Ladder](/ladder/matrix.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.